### PR TITLE
[TorchTidy] Reland #81941 Add pattern to detect if bias is enabled in conv2d followed by batchnorm2d

### PR DESCRIPTION
**Pull request by davidchencsl**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #82261
* #82248
* __->__ #82421

### CODE REVIEW

**Comment by davidchencsl** [comment]
@davidchencsl has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D38254079).

**Comment by davidchencsl** [comment]
@davidchencsl has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D38254079).

**Comment by davidchencsl** [comment]
@davidchencsl has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D38254079).

**Comment by davidchencsl** [comment]
@davidchencsl has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D38254079).

**Comment by osalpekar** [comment]
@davidchencsl #81941 is already reverted, see https://github.com/pytorch/pytorch/commit/2fe73164b67113baa480d70df3feff55d48d0c53. I'm assuming that was the intent of this PR? In that case, we shouldn't merge this PR and should close it.

**Comment by davidchencsl** [comment]
> @davidchencsl #81941 is already reverted, see [2fe7316](https://github.com/pytorch/pytorch/commit/2fe73164b67113baa480d70df3feff55d48d0c53). I'm assuming that was the intent of this PR? In that case, we shouldn't merge this PR and should close it.

This PR was to fix the issue with the reverted PR and reland it

**Review by robieta — APPROVED** [review]
LGTM
